lib/papyrus/life_cycle.rb

```diff
@@ -9,13 +9,11 @@ module LifeCycle
 
       after_commit do
         if previously_persisted?
-          Papyrus.event(:destroy, self)
-        elsif previously_new_record?
-          Papyrus.event(:create, self)
-          Papyrus.event(:save, self)
+          Papyrus.event(:destroy, self, Papyrus.config.default_params(:destroy, self))
```

Comment:
I don't find this more readable, less so, please revert this.
It's a good find on the default_params, lets put it in Papyrus.event method instead.

Reply:
adding the default params config to each line was less readable ,to me atleast.
I moved the default params to inside the event.